packages/rfw/test/text_test.dart

```diff
@@ -193,7 +193,9 @@ void main() {
     expect(parseDataFile('{ "a": -1e-16 }'), <String, Object?>{ 'a': -0.0000000000000001 });
     expect(parseDataFile('{ "a": -1e-17 }'), <String, Object?>{ 'a': -0.00000000000000001 });
     expect(parseDataFile('{ "a": -1e-18 }'), <String, Object?>{ 'a': -0.000000000000000001 });
-    expect(parseDataFile('{ "a": -1e-19 }'), <String, Object?>{ 'a': -0.0000000000000000001 });
+    expect(parseDataFile('{ "a": infinity }'), <String, Object?>{ 'a': double.infinity });
+    expect(parseDataFile('{ "a": negativeInfinity }'), <String, Object?>{ 'a': double.negativeInfinity });
+    expect(parseDataFile('{ "a": nan }')['a'], isNaN);
```

Comment:
nit: For the sake of consistency, should this particular test be

expect(..., <String, Object?> { 'a': double.nan });

Reply:
that would fail, because `double.nan == double.nan` is false